code/modules/projectiles/updated_projectiles/projectile.dm

```diff
@@ -359,7 +359,7 @@
 /obj/structure/window/get_projectile_hit_chance(obj/item/projectile/P)
 	if(P.ammo.flags_ammo_behavior & AMMO_ENERGY)
 		return FALSE
-	else if(!(flags_atom & ON_BORDER) || (P.dir & dir) || (P.dir & reverse_direction(dir)))
+	else //if(!(flags_atom & ON_BORDER) || (P.dir & dir) || (P.dir & reverse_direction(dir)))
```

Comment:
Are you commenting out this code because you know what's going on here, or because it empirically works as intended after removing it?

Reply:
Commenting out the code in case it needs to be reinstated or parts of it do. Thus far getting rid of it seems to have achieved the desired effect without any undesired side effects that I've noticed thus far.

Reply:
Test if bullets can pass through tiles with directional windows that shouldn't be blocking them with this fix.